.github/workflows/docker-build-static.yml

```diff
@@ -473,6 +473,20 @@ jobs:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
+      - name: Login to GitHub Container Registry
+        if: github.event_name != 'pull_request'
+        uses: docker/login-action@v2
+        with:
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      # - name: Login to nitnelave GitHub Container Registry
```

Comment:
That one should not be needed, the login above should be enough. Maybe it only works with nitnelave though?

Reply:
okay I updated to only login using nitnelave, not sure if it'll work but you can try it out